addon/webextension/content-script.js

```diff
@@ -41,11 +47,12 @@ function findAndReplace(wordList) {
   document.querySelectorAll(".donotdelete").forEach((node) => {
     const hoverEle = document.createElement("span");
     hoverEle.innerHTML = `
-      Mr. Robot something something.
-      <br>
-      <a href="http://www.mozilla.org" target="_blank">
-        Learn more
-      </a>`;
+    Can you trust your perceptions?
+    You chose this... a reminder of the forces at work in your world.
+    If you no longer wish to peer through the looking glass, you can
+    <br/><a href="${SUPPORTURL}" target="_blank">
```

Comment:
Re: `target="_blank"`, not sure if we want to add `rel="noopener noreferrer"`, per https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/

Reply:
done!